### ISSUE TITLE
[doc] missing torch.pdist entry

### ISSUE THREAD

**Issue by SsnL** [pytorch/pytorch#49426]
## 📚 Documentation
It is mentioned in `TorchScript Builtins`, but has no doc entry.

**Comment by anjali411** [comment]
`pdist` is currently exposed in both `torch.*` and `torch.nn.functional.*`. I am not sure if that's intended

**Comment by albanD** [comment]
@anjali411 that is true for a few of them as the functions that were historically in `torch.nn.functional`, when lowered to c++ (and added to native_functions.yaml) usually end up in the `torch.` namespace. So the functional version just calls back into that.

**Comment by anjali411** [comment]
Okay then I think we should move the doc entry to torch page and link that under `torch.nn.functional.pdist` as "alias of `torch.pdist`"?

**Comment by mruberry** [comment]
I suggest we leave the docs as they are now for now. Having undocumented ops is OK if, for example, we don't think these operations should be exposed in the torch namespace. Adding aliases for them to their torch.nn.functional equivalents will just clutter the docs and cause confusion about which operator to use.
